Clean up everything

natope opened the pull request.

natope committed:
Create URL preprocessing.ipynb

natope committed:
Fix import cells in URL preprocessing.ipynb

natope committed:
Include get_doc_text in html_preprocessing.py

natope committed:
Create TF-IDF Ranking.ipynb

natope committed:
Create TF-IDF_smaller_sample.ipynb

natope committed:
Update TF-IDF_smaller_sample.ipynb

Include classification report

natope committed:
work-in-progress

natope committed:
Create URL_Analysis.ipynb

URL exploration and analysis to ensure the correct steps taken for URL collection.

natope committed:
Create collect_documents.py

natope committed:
Create collect_documents.py

natope committed:
Create preprocess_documents.py

natope committed:
Fix dataset folder

natope committed:
rename Document Collection to document_collection

natope committed:
Create collect_documents_clean.py

natope committed:
rename folder Reference Documents to reference_documents

natope committed:
rename folder Supporting Documents to supporting_documents

natope committed:
Creatte collect_documents_clean.py

natope committed:
Create dataset_filtering.py

natope committed:
Update dataset.py

natope committed:
Create collect.py for amsterdam.nl

natope committed:
remove unused files from template

natope committed:
Create collect.py

natope committed:
Create rivm/collect.py

natope committed:
Create ggd/collect.py

natope committed:
Delete wget files folder

natope committed:
delete template documennts

natope committed:
delete URL Scraping folder

which contained only rijksoverheid collection

natope committed:
rename ranking folder to retrieval

natope committed:
Update folder structure for retrieval

natope committed:
Remove rijksoverheid document collection notebooks

natope committed:
remove documents from template

natope committed:
Update dataset.py

natope committed:
add reference_documents/__init__.py

natope committed:
Fix apply_url_fetching function in collect_documents_clean.py

natope committed:
Create Reference_collection.ipynb

Reference_collection.ipynb to test whether the implementation of document collection worked properly

natope committed:
Create settings.json

natope committed:
Update dataset.py

self.pdf_content and self.html_content to just self.content

natope committed:
Update dataset.py

- Add collect_urls to class QuestionAnswer
- add try-except to fetch_url to class ReferenceURL and store exceptions in self.exception

natope committed:
Update filter_by_domains in class QuestionAnswer in dataset.py

- instead of replacing url with an empty '', remove it in filter_by_domains

natope committed:
Update dataset - class QuestionAnswer

- fix self.urls

natope committed:
Create Collect_references.ipynb

natope committed:
Include create_dataset in dataset.py

natope committed:
Add filetring options to create_dataset

natope committed:
Update dataset.py

natope committed:
Fix filter_by_domains

natope committed:
fix filter_by_domains

natope committed:
Update Collect_references.ipynb

- test create_dataset

natope committed:
Add new placeholder files

natope committed:
add clean_url

natope committed:
Update Collect_references.ipynb

natope committed:
Update dataset.py

natope committed:
Update collect_documents_clean.py

natope committed:
Add saving functions

natope committed:
apply saving functions

natope committed:
remove print

natope committed:
move imports to one cell

natope committed:
add http in front of urls that don't have it

natope committed:
keep track of question-url in terms of pairs

natope committed:
Save collected data

natope committed:
Delete Reference_collection.ipynb

natope committed:
update collection waiting time

natope committed:
Create collection_preprocessing.py

natope committed:
Create random_retrieve.py

natope committed:
Create evaluate.py

natope committed:
Delete collect_documents.py

natope committed:
Add collect.py for ggd and rivm

natope committed:
Create bm25.ipynb

natope committed:
Update read_data.py

natope committed:
Update tfidf.py

natope committed:
Create bm25.py

natope committed:
Update collect_documents_clean.py

natope committed:
Update collection_functions.py

natope committed:
Rename bm25.ipynb to TF-IDF_Random_test.ipynb

natope committed:
Fix amsterdam collection

natope committed:
Delete unused notebooks

natope committed:
Move drafts to a Drafts folder

natope committed:
Update .gitignore

ignore .csv files

natope committed:
Update .gitignore

Ignore all .pickle files

natope committed:
Remove unused notebooks

natope committed:
Create TF-IDF.ipynb

natope committed:
Create BM25.ipynb

natope committed:
Create Random_retr.ipynb

natope committed:
Delete TF-IDF_Random_test.ipynb

natope committed:
Move URL Analysis to URL collection folder

natope committed:
Add possibility for keyboard interruption in URL collection

natope committed:
Update bm25.py

natope committed:
Update evaluate.py

natope committed:
Update read_data.py

natope committed:
Create Collect_references.ipynb

natope committed:
Create Collect_amsterdam.ipynb

natope committed:
Create mT5.ipynb

natope committed:
Create EDA_V1.ipynb

natope committed:
Move URL_Analysis to EDA

natope committed:
Include EDA notebooks

natope committed:
Remove unnecessary functions

natope committed:
Remove unnecessary function

natope committed:
Delete preprocess_documents.py

natope committed:
Delete Collection-draft

natope committed:
add random retrieval recall calculation

natope committed:
Update random_retrieve.py

natope committed:
Update Collect_amsterdam.ipynb

natope committed:
Remove unused files

natope committed:
Update collect_documents_clean_clean.py

natope committed:
Create preprocess_ranking_results.py

natope committed:
Update Random_retr.ipynb

natope committed:
Update README.md

natope committed:
Update README.md

natope committed:
Update README.md

natope committed:
Update README.md

natope committed:
Update README.md

natope committed:
Update README.md

igornishka commented on .DS_Store:
can you remove all the .DS_store files
`./notebooks/.DS_Store
./notebooks/notebooks-clean/.DS_Store
./.DS_Store
`

For things you don't want delete locally, just remove from git history, you can do `git rm --cached $file`

You can also add it to the gitignore

igornishka commented on .vscode/settings.json:
Same for the .vscode folder - can you remove and add .vscode/* to gitignore?
`git rm --cached .vscode/settings.json` to preserve the file if you need it

igornishka commented on notebooks/notebooks-clean/EDA/EDA_V1.ipynb:
```diff
@@ -0,0 +1,2122 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_all = pd.read_csv('/Users/natalipeeva/Desktop/Data/questions.csv')"
```
hardcoded path

You can grep to see them all:
```
(env) (base) iva@iva-XPS-15-9500:~/projects/gemeente/InternshipAmsterdam/Automatic-Answering-of-City-Council-Questions$ grep -r '/Users/natalipeeva/'
src/read_data/read_data.py:src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/', 'src')
src/document_collection/supporting_documents/collection_clean/collection_functions.py:src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Draft/', 'src')
src/document_collection/supporting_documents/collect_documents_clean_clean.py:src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Draft/', 'src')
src/document_collection/supporting_documents/amsterdam_nl/collect.py:src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Draft/', 'src')
src/document_collection/reference_documents/collect_documents_clean.py:src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/', 'src')
notebooks/notebooks-clean/EDA/EDA_V1.ipynb:    "df_all = pd.read_csv('/Users/natalipeeva/Desktop/Data/questions.csv')"
notebooks/notebooks-clean/EDA/EDA_V1.ipynb:    "df_trans_links = pd.read_csv('/Users/natalipeeva/Desktop/Data/Questions_links_translations.csv')"
notebooks/notebooks-clean/EDA/URL_Analysis.ipynb:    "questions = pd.read_csv(open('/Users/natalipeeva/Desktop/Clean work/questions.csv', 'r'))\n",
notebooks/notebooks-clean/EDA/Type of questions.ipynb:    "df_all = pd.read_csv('/Users/natalipeeva/Desktop/Data/questions.csv')"
notebooks/notebooks-clean/EDA/Type of questions.ipynb:    "df_trans_links = pd.read_csv('/Users/natalipeeva/Desktop/Data/Questions_links_translations.csv')"
notebooks/notebooks-clean/URL_Collection/Collect_references.ipynb:    "src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/', 'src')\n",
notebooks/notebooks-clean/URL_Collection/Collect_references.ipynb:    "data = create_dataset('/Users/natalipeeva/Desktop/Clean work/questions.csv', # add clean urls\n",
notebooks/notebooks-clean/URL_Collection/Collect_amsterdam.ipynb:    "src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/', 'src')\n",
notebooks/notebooks-clean/Ranking/TF-IDF.ipynb:    "src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/', 'src')\n",
notebooks/notebooks-clean/Ranking/TF-IDF.ipynb:    "collected, questions =  read_urls_questions('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/notebooks/notebooks-clean/URL_Collection/reference_urls_collected.csv',\n",
notebooks/notebooks-clean/Ranking/TF-IDF.ipynb:    "                                            '/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/notebooks/notebooks-clean/URL_Collection/questions_updated_urls.csv',\n",
notebooks/notebooks-clean/Ranking/Random_retr.ipynb:    "src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/', 'src')\n",
notebooks/notebooks-clean/Ranking/Random_retr.ipynb:    "collected, questions =  read_urls_questions('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/notebooks/notebooks-clean/URL_Collection/reference_urls_collected.csv',\n",
notebooks/notebooks-clean/Ranking/Random_retr.ipynb:    "                                            '/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/notebooks/notebooks-clean/URL_Collection/questions_updated_urls.csv',\n",
notebooks/notebooks-clean/Ranking/BM25.ipynb:    "src_dir = os.path.join('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/', 'src')\n",
notebooks/notebooks-clean/Ranking/BM25.ipynb:    "collected, questions =  read_urls_questions('/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/notebooks/notebooks-clean/URL_Collection/reference_urls_collected.csv',\n",
notebooks/notebooks-clean/Ranking/BM25.ipynb:    "                                            '/Users/natalipeeva/Documents/GitHub/Automatic-Answering-of-City-Council-Questions/notebooks/notebooks-clean/URL_Collection/questions_updated_urls.csv',\n",

natope committed:
Update README.md

igornishka commented on src/document_collection/reference_documents/collect_documents_clean.py:
mind renaming all the _clean(_clean) files?

```
(env) (base) iva@iva-XPS-15-9500:~/projects/gemeente/InternshipAmsterdam/Automatic-Answering-of-City-Council-Questions$ find src -name '*clean*'
src/document_collection/supporting_documents/collection_clean
src/document_collection/supporting_documents/collect_documents_clean_clean.py
src/document_collection/reference_documents/collect_documents_clean.py
(env) (base) iva@iva-XPS-15-9500:~/projects/gemeente/InternshipAmsterdam/Automatic-Answering-of-City-Council-Questions$ find notebooks -name '*clean*'
notebooks/notebooks-clean
```

igornishka reviewed (commented):
Other than that

- [ ] plenty of unused imports (you can run flake8 yourself or I can send the list)
- [ ] 

I'll slowly run stuffs and update

natope committed:
Update collect_documents_clean_clean.py

natope committed:
Create combine_collected.py

natope committed:
Update collect.py

natope committed:
Update collect_documents_clean.py

natope committed:
Create combine_all_collected.ipynb

natope committed:
Update Collect_references.ipynb

natope committed:
Create Collection_amsterdam_individual.ipynb

natope committed:
Update Collect_amsterdam.ipynb

natope committed:
Update TF-IDF.ipynb

UPdate paths

natope committed:
Update BM25.ipynb

update paths - add elastic search instruction

natope committed:
Create Combine_collected.ipynb

natope committed:
Update paths in reference collection script

natope committed:
Update paths in supporting document collection

natope committed:
update paths in read data

natope committed:
prepare for GPL pre-training (not needed)

natope committed:
Create .gitkeep

natope committed:
add data folders

natope committed:
remove path from function

natope committed:
update collection after path removal from function

natope committed:
Create Ranking_Comparison_TFIDF_Random_BM25.ipynb

- Creates passages from collection
- Preprocesses data
- Compares tfidf, random, bm25
- in terms of IR metrics & ROUGE
- saves data

natope committed:
Adds functions needed for ranking comparison

natope committed:
Update ranking

applicable for the bigger collection too

natope committed:
Create Ranking_All_Questions.ipynb

natope committed:
Create mT5_TFIDF_paragraphs_preprocessed_questions_22_06_2023.ipynb

natope committed:
Update Files
